### PR TITLE
Bring back pro components to charts

### PR DESCRIPTION
**Pull request by cosmic-flood**

### CODE REVIEW

**Comment by cosmic-flood** [comment]
no kafka_broker_list env parameter in the example and values.yaml

**Comment by cosmic-flood** [comment]
`CLICKHOUSE_KAFKA_HOSTS` missed

@dsun0720

File: charts/featbit/templates/api-deployment.yaml
Comment (wss-rbrennan) [inline charts/featbit/templates/api-deployment.yaml:1]: IS_PRO needs to be set when `isPro: true` is set in values .  This is evaluated when `API\Setup\ServicesRegister.cs` makes calls to `AddInfrastructureServices` in `Infrastructure\ConfigureServices.cs`

File: charts/featbit/templates/eval-server-deployment.yaml
Comment (wss-rbrennan) [inline charts/featbit/templates/eval-server-deployment.yaml:1]: IS_PRO needs to be set when `isPro: true` is set in values . This is evaluated in `API\Setup\ServicesRegister.cs` and determines whether Kafka or Redis is used as the message queue

File: charts/featbit/templates/da-server-deployment.yaml
Comment (wss-rbrennan) [inline charts/featbit/templates/da-server-deployment.yaml:1]: IS_PRO needs to be set when `isPro: true` is set in values . This is evaluated by the da server to determine whether to use clickhouse or mongodb as the datastore, and to query accordingly

**Review by wss-rbrennan — COMMENTED** [review]
Other than comments regarding the IS_PRO env var needing to be set this LGTM.  Passes linting, and installs without issue.  Adding the missing env vars is key though.

**Comment by cosmic-flood** [comment]
> Other than comments regarding the IS_PRO env var needing to be set this LGTM. Passes linting, and installs without issue. Adding the missing env vars is key though.

Hi @wss-rbrennan If I understand your question correctly, I believe those details have already been included in the charts, as the images below show:

image

image

image

image

**Review by wss-rbrennan — APPROVED** [review]

**Review by wss-rbrennan — APPROVED** [review]

**Review by deleteLater — APPROVED** [review]
LGTM

**Pull request merged**